### ISSUE TITLE
Experiment Tracking Metrics is not shown on the kedro-viz UI on Window 10

### ISSUE THREAD

**Issue by noklam** [kedro-org/kedro-viz#822]
## Description
When running experiment tracking on Window 10, the metrics do not show up properly.
User @Zemeio reported an issue on Discord that experiment tracking is not working. I can also reproduce the issue on my Windows machine.

The discussion thread in Discord can be found [here](https://discord.com/channels/778216384475693066/961381812020596836). 

## Context
How has this bug affected you? What were you trying to accomplish?

## Steps to Reproduce

A demo repo can be found [here](https://github.com/noklam/kedro_viz_demo)

1. `kedro run` to generate the experiment data
2. `kedro viz`

## Expected Result
Metrics should be shown on the UI

## Actual Result
![image](https://user-images.githubusercontent.com/18221871/162794483-67eacc15-1ba7-467b-9ca0-c44b2fe6128d.png)
The files exists in the filesystem but the number is not shown on UI.

```
-- If you received an error, place it here.
```

```
-- Separate them if you have more than one.
```

## Your Environment
Include as many relevant details as possible about the environment you experienced the bug in:

* Web browser system and version:
* Operating system and version: Window 10
* NodeJS version used (if relevant):
* Kedro version used (if relevant): kedro 0.18.0, kedro-viz 4.4.0
* Python version used (if relevant): 3.8.5

## Checklist
- [ ] Include labels so that we can categorise your issue

**Comment by tynandebold** [comment]
This has been fixed in #809 but hasn't been released yet.

**Comment by noklam** [comment]
Got it, thanks @tynandebold.
